test/fastboot-test.js

```diff
@@ -101,16 +101,26 @@ describe("FastBoot", function() {
       });
   });
 
-  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
+  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function(done) {
     var fastboot = new FastBoot({
       distPath: fixture('basic-app')
     });
 
-    return fastboot.visit('/', {
-      destroyAppInstanceInMs: 5
-    })
+    // delaying `visitRoute` to forcefully destroy app instance
+    let originalVisitRoute = fastboot._app.visitRoute;
```

Comment:
added `long-resolving-app` with application route that has model which resolves in 10 seconds, that revealed 2 problems:
* timed destroy could be fired too early before app instance created (so ended up not destroying app)
* when app destroyed before `instance.visit` is called, it was throwing error for setting property for destroyed object (which is still thrown but it is superseded by `App instance was forcefully destroyed ...`).

both of this issues handled now